test/test.conf

```diff
@@ -1,3 +1,3 @@
 [func_test]
-gfs_host = gfshost
-gfs_volname = test
+gfs_host = 192.168.1.2
```

Comment:
I properly changed my `/etc/hosts` at this point. I do think that could benefit later contributors though.